src/data/pokemon/cry_ids.h

```diff
@@ -0,0 +1,140 @@
+#define HOENN_MON_SPECIES_START 277
+
+static const u16 sHoennSpeciesIdToCryId[] =
+{
+    [SPECIES_TREECKO - HOENN_MON_SPECIES_START] = 273,
```

Comment:
Perhaps define enums for the Hoenn cry IDs?

Reply:
Now enums are in include/constants/hoenn_cries.h